src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/elinks/CheckForSearchV2UsersIntegrationTest.java

```diff
@@ -188,7 +188,7 @@ void shouldReturn200WhenUserProfileRequestedSscsAppointmentExpiredIacSscsAuthAct
                 MediaType.valueOf(V2.MediaType.SERVICE));
         var profiles = (List<Map<String, String>>) response.get("body");
         if (("BBA3").equals(serviceCode)) {
-            assertEquals(0, profiles.size());
+            assertEquals(1, profiles.size());
```

Comment:
Can we confirm in the integration tests data that we have a scenario to test mapping where

1.  mapped via personcode and apponitment_id is null (i.e. data should be found)
2. mapped via  personcode and appointment_id is set to **active** appointment (i.e. data should be found)
3. mapped via  personcode and appointment_id is set to **expired** appointment (i.e. data should not be found)

From my quick (possibly incorrect) review of data in 'src/integrationTest/resources/db/testmigration/V1_74__elinks_insert_user_profile.sql':  I assume this assert should probably return 0. as I think the appointment has expired.